src/d3a/d3a_core/simulation.py

```diff
@@ -169,7 +170,11 @@ def _init(self, slot_length_realtime, seed, paused, pause_after, redis_job_id):
             log.info("Random seed: {}".format(random_seed))
 
         self.area = self.setup_module.get_setup(self.simulation_config)
+        # TODO: discuss whether the following circular reference is a good idea (probably not)
+        #  it is needed for being able to update the whole grid_tree from the devices
+        self.global_objects = GlobalStatistics(self.area)
```

Comment:
```suggestion
        self.global_stats = GlobalStatistics(self.area)
```

minor but since you renamed the class